### PR TITLE
Fix tvm build

### PR DESCRIPTION
**Pull request by Ailurus1**
Fixed the build of TVM with automatically found LLVM installed using apt

### CODE REVIEW

File: cm-mlops/script/get-ml-model-resnet50-tvm/_cm.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
   ],
   "tags": [
     "get",
+    "raw",
```
Comment (arjunsuresh) [inline cm-mlops/script/get-ml-model-resnet50-tvm/_cm.json:28]: Actually we use "raw" tag to distinguish between tvm models (or any other optimized ones) and the original ones by putting "raw" tag only to the original ones. All other changes are good to be approved. Thanks for them.

**Review by arjunsuresh — COMMENTED** [review]
Except the "raw" tag, all others are good to be approved. The issue with "raw" tag is fixed in the other pending PR.

**Comment by arjunsuresh** [comment]
recheck

**Comment by arjunsuresh** [comment]
recheck

**Comment by arjunsuresh** [comment]
recheck

**Comment by arjunsuresh** [comment]
Hi @Ailurus1, CLA checks are now fine. Would you like to continue working on this PR or shall we merge this?

**Comment by Ailurus1** [comment]
Hello, @arjunsuresh. I think it can be merged

**Review by arjunsuresh — APPROVED** [review]

**Comment by arjunsuresh** [comment]
Thank you @Ailurus1. I have merged now.

**Pull request merged**